src/components/custom-field-input-currency/custom-field-input-currency.test.jsx

```diff
@@ -35,7 +35,7 @@ describe('CustomFieldInputCurrency', () => {
   describe('prop-forward API', () => {
     it('forwards all props accepted by CustomFieldInputText on Object keys', () => {
       const excludedNumberProps = ['inputRef', 'max', 'min', 'onKeyUp', 'onKeyDown', 'step', 'onBlur', 'onFocus',
-        'type', 'readOnly', 'icon', 'onChange'];
+        'type', 'readOnly', 'icon', 'onChange', 'onClick'];
```

Comment:
Just to capture our talk: we probably want to start deprecating the "prop-forward API" tests since they tend to be pretty brittle?